responses/es/05_image-works-url.md

```diff
@@ -0,0 +1,42 @@
+## Paso 6: Añade un enlace de perfil
+
+¡Tu imagen tiene muy buen aspecto! Ahora querrás añadir enlaces a los impresionantes proyectos de tu portafolio. Los enlaces ayudan a crear contexto cuando te comunicas en las propuestas (o _issues_) y solicitudes de extracción (o _pull requests_).
+
+Puedes enlazar un [sitio web](https://github.com/), un [repositorio](https://github.com/github/training-kit), o incluso una [línea de código](https://github.com/github/training-kit/blob/master/resources/learning-path/index.html#L32). Para crear un enlace, coloca el texto que quieres mostrar entre corchetes, y la URL a continuación entre paréntesis.
+
+```
+[GitHub](http://github.com)
+```
+
+## :keyboard: Actividad: Añade un enlace
+
+{% if preferences.gitTool == 'cli' %}
+1. En tu shell, asegúrate de que estás en la rama `add-images-links`:
+      ```shell
+      git status
+      ```
+1. Si _no_ lo estás, entonces cámbiate a esa rama:
+      ```shell
+      git checkout add-images-links
+      ```
+1. En tu editor de textos, abre el archivo `03-links.md`, en el directorio `_includes`. Replace the filler text with a link to your GitHub profile (or anywhere else).
```

Comment:
Una frase necesita una traducción: "Replace the filler text with a link to your GitHub profile (or anywhere else)."